Sources/CovidCertificateSDK/TrustList/TrustListManager.swift

```diff
@@ -183,14 +183,24 @@ class TrustListUpdate {
 
     private func startForceUpdate() {
         internalQueue.sync {
-            let error = synchronousUpdate(ignoreLocalCache: true)
-            // Only reset lastError if synchronousUpdate was successful
-            if error == nil {
-                operationQueue.addOperation {
-                    self.lastError = nil
-                }
-            }
+            lastError = synchronousUpdate(ignoreLocalCache: true)
             forceUpdateOperation = nil
         }
     }
+
+    // TODO: Read value from config request
+    private static let allowedServerTimeDiff: TimeInterval = 60 * 10 // 10 minutes
```

Comment:
Let's set it to 2h for now and then expose it as a configurable parameter, which the users of the SDK can overwrite.